components/recipe_display.js

```diff
@@ -0,0 +1,95 @@
+                                <Typography>
+                                    {this.props.recipe.description}
+                                </Typography>
+                            </CardContent>
+                        </Card>
+                        <Card>
+                            <CardHeader title="Tags" />
+                            <CardContent>
+                                {this.props.recipe.tags.map(
+                                    tag => <Chip key={tag} label={tag} />
+                                )}
+                            </CardContent>
+                        </Card>
+                        <Card>
+                            <CardHeader title="Ingredients" />
+                            <CardContent>
+                                <List>
+                                    {this.props.recipe.ingredients.map( ingredient => 
+                                        <ListItem key={ingredient}>
+                                            <ListItemText 
+                                                primary={ingredient.quantity + " " + ingredient.unit + " " + ingredient.name}
+                                            />
+                                        </ListItem>
+                                    )}
+                                </List>
+                            </CardContent>
+                        </Card>
+                    </Box>
+                    <Box>
+                        <Card>
+                            <CardHeader title="Equipment" />
+                            <CardContent>
+                                <List>
+                                    {this.props.recipe.equipment.map( equipment =>
+                                        <ListItem key={equipment}>
+                                            <ListItemText primary={equipment} />
+                                        </ListItem>
+                                    )}
+                                </List>
+                            </CardContent>
+                        </Card>
+                        <Card>
+                            <CardHeader title="Directions" />
+                            <CardContent>
+                                <List>
+                                    {this.props.recipe.directions.map( direction =>
+                                        <ListItem key={direction}>
+                                            <ListItemIcon>
+                                                <Checkbox />
+                                            </ListItemIcon>
+                                            <ListItemText primary={direction} />
+                                        </ListItem>    
+                                    )}
+                                </List>
+                            </CardContent>
+                        </Card>
+                    </Box>
+                </Box>
+            </ThemeProvider>
```

Comment:
This JSX is extremely deeply nested, which makes it hard to read. I'd suggest breaking it up into multiple components in the same file. Perhaps add TagsCard, DescriptionCard, EquipmentList, DirectionsList, etc.